conf/evolutions/default/85.sql

```diff
@@ -0,0 +1,14 @@
+# --- !Ups
+
+ALTER TABLE `users`
+ADD COLUMN `date_created` DATETIME NULL AFTER `last_login`;
+ADD COLUMN `created_by` INT NOT NULL AFTER `date_created`;
+
```

Comment:
Kind of, the line 

ADD COLUMN date_created DATETIME NULL AFTER last_login

should end with a comma not ;